src/api/api.tsx

```diff
@@ -35,10 +42,48 @@ export default class API {
     });
   }
 
-  addPdfs(pdfs: FormData) {
-    return this.fetchAdvanced(this.addPdfsURL(), {
+  addPdfs(pdf: FormData) {
+    return this.fetchAdvanced(this.addPdfURL(), {
       method: 'POST',
-      body: pdfs,
+      body: pdf,
+    });
+  }
+
+  getApplicants() {
+    return this.fetchAdvanced(this.getApplicantsURL(), {
+      method: 'GET',
+    }).then((res) => {
+      let applicants = createApplicant(res);
```

Comment:
The responses should be named properly with the correct type:
Example:
- `res` should be called `applicant` with the type `Applicant`

Reply:
Have you considered that we not receiving one applicant but many applicants do to the name of the function?
Therefore the name 'applicant' for the variable 'res' would be 'applicants' which doubles with the variable name in line 56. In addition the type of 'res' is not Applicant but a json string.